java/test/android/controller/src/main/java/com/zeroc/testcontroller/ControllerApp.java

```diff
@@ -15,6 +15,10 @@
 import com.zeroc.Ice.Logger;
 import com.zeroc.Ice.Time;
 
+import java.util.List;
+import java.util.Locale;
+import java.io.Writer;
```

Comment:
We used to have a `*` import, seems it was removed by checkstyle changes, and we didn't notice because we are not building Android on CI.